doc/SETUP.md

```diff
@@ -52,14 +52,9 @@ git clone https://github.com/jasongin/nvs "$env:NVS_HOME"
 After installation, running just `nvs` from the current shell or any new Command Prompt or PowerShell then invokes either the `nvs.cmd` or `nvs.ps1` script accordingly.
 
 ### Git Bash on Windows
-NVS can work in Git Bash on Windows (the bash tools installed by [Git](https://git-scm.com/) for Windows), though it requires some manual configuration:
+NVS can work in Git Bash on Windows (the bash tools installed by [Git](https://git-scm.com/) for Windows) automatically.
```

Comment:
If I understand correctly, it still requires running `nvs install` from a git bash shell, right? I think "automatically" implies it should work without that - as in just installing the MSI on Windows.

Reply:
true. I'll correct it